ui/ui.go

```diff
@@ -96,29 +96,52 @@ func (ui *WriterUI) PrintTable(table Table) {
 	}
 }
 
-func (ui *WriterUI) AskForText(label string) (string, error) {
+func (ui *WriterUI) AskForText(opts TextOpts) (string, error) {
 	var text string
 
-	err := interact.NewInteraction(label).Resolve(&text)
-	if err != nil {
-		return "", fmt.Errorf("Asking for text: %s", err)
+	for {
+		text = opts.Default
```

Comment:
This should be outside for loop.

Reply:
Why I keep this inside the for loop is supposed in the first iteration instead of hitting enter user gave input on the terminal and suppose it's invalid then if we don't set the chosen = opts.Default again then the invalid input that the user-provided will become the Default now and I think we should not change our default in such scenarios.